torchsummary/torchsummary.py

```diff
@@ -98,7 +98,9 @@ def hook(module, input, output):
         summary_str += line_new + "\n"
 
     # assume 4 bytes/number (float on cuda).
-    total_input_size = abs(np.prod(sum(input_size, ()))
+    # to handle the case of multi-input: prod(input1) + prod(input2) + ...
+    n_input_size = np.array([np.prod(i) for i in input_size]).sum() if isinstance(input_size, list) else np.prod(input_size)
```

Comment:
we don't need to check if `input_size` is a list because that is normalized here:https://github.com/sksq96/pytorch-summary/blob/4852f2efb1ccfa9cfda1a49519a633d21a170fe7/torchsummary/torchsummary.py#L54-L56

Reply:
I wrote this long time ago, but I think I added this check so that the code works properly for the single-input cases too. If the its a single-input, return the product of input dims, if its multi-input (list of inputs), return prod(input1) + prod(input2) + ...